challenge-08/challenge-08.js

```diff
@@ -3,38 +3,43 @@ Declare uma variável chamada `sum` e atribua a ela uma função chamada
 `calculateSum`. A função deve receber dois parâmetros e retornar a soma
 desses parâmetros.
 */
-// ?
+var sum = function calculateSum(a,b) {
+    return a+b;
+};
 
 /*
 Invoque a função criada acima, passando dois números que serão somados, e mostre
 o resultado no console, com a frase:
 "A soma de [VALOR 1] e [VALOR2] é igual a [RESULTADO]."
 */
-// ?
+
+console.log('A soma de a e b é igual a ' +sum(1,2));
```

Comment:
-1
Tem que mostrar o valor no lugar de `a` e `b` :)
Outra coisa: sempre que for mostrar mais de um valor no console, separe como se fossem parâmetros. Não concatene com `+`, senão o resultado pode não ser o esperado :)